lib/game_state.rb

```diff
@@ -1,36 +1,34 @@
 module TicTacToe 
-  class GameState
```

Comment:
I would still add an inner module/class within the TicTacToe namespace to be more explicit where these methods are coming from. In code, it's hard to tell as they're called using `TicTacToe.method_name`